.ci/infra/terraform/main.tf

```diff
@@ -0,0 +1,481 @@
+#   - A second existing billing account where charges can be applied (used only for TestAccProject_billing)
+#   - An existing billing account where subaccounts can be created
+#   - A BeyondCorp subscription on the organization
+#
+# After applying this configuration:
+#   - (Internal only) Enable stubbed calls for GKE MultiCloud resources
+#   - (Internal only) Verify ownership of hashicorptest.com for new service account
+#   - Enable Media CDN
+#   - Enable Game Services
+#   - Enable Access Boundary permissions
+#   - Deploy "Hello World" app: https://cloud.google.com/appengine/docs/flexible/go/create-app
+#     ```
+#     gcloud components install app-engine-go
+#     git clone https://github.com/GoogleCloudPlatform/golang-samples
+#     cp -r golang-samples/appengine_flexible/helloworld ./.
+#     cd helloworld
+#     gcloud app deploy --project=<project>
+#     ```
+#   - Create repo for "Hello World" function: https://cloud.google.com/source-repositories/docs/deploy-cloud-functions-version-control
+#     ```
+#     gcloud source repos create cloudfunctions-test-do-not-delete --project=<project>
+#     gcloud source repos clone cloudfunctions-test-do-not-delete --project=<project>
+#     cd cloudfunctions-test-do-not-delete
+#     curl https://raw.githubusercontent.com/GoogleCloudPlatform/magic-modules/main/mmv1/third_party/terraform/utils/test-fixtures/cloudfunctions/http_trigger.js > index.js
+#     git add .
+#     git commit -m "Initial commit"
+#     git push origin main
+#     git checkout -b master
+#     git push origin master
+#     ```
+#   - Enable Multi-Tenancy
+#     ```
+#     curl --header "Authorization: Bearer $(gcloud auth print-access-token -q)" --header "X-Goog-User-Project: <project>" -X POST https://identitytoolkit.googleapis.com/v2/projects/<project>/identityPlatform:initializeAuth
+#     curl --header "Content-Type: application/json" --header "Authorization: Bearer $(gcloud auth print-access-token -q)" --header "X-Goog-User-Project: <project>" -X PATCH https://identitytoolkit.googleapis.com/admin/v2/projects/<project>/config?updateMask=multiTenant -d '{"multiTenant": {"allowTenants": true}}'
+#     ```
+#   - Add Group Admin role to new service account in the Google Workspace Admin Console: https://admin.google.com/ac/roles
+#   - Add a new test user in the Google Workspace Admin Console: https://admin.google.com/ac/users
+#   - Create a support@ group in the Google Workspace Admin Console, add new service account as a member, and make it an owner
+#
+# Quotas that will need to be adjusted to support all tests:
+#   - Project quota for the new service account
+#   - Project quota for the billing account
+#   - Compute Networks quota in us-central1
+#   - CPUS quota in us-central1
+#   - Filestore Basic SSD quota in us-central1
+#   - Compute Disk IOPS quota  in us-central1
+#   - AlloyDB cluster quota in us-central1
+#   - Cloud Workstation cluster quota in us-central1
```

Comment:
Does it make sense if we add a readme file for this part?

Reply:
Haha yes, that's a much better idea. I think this made sense to me when it was a much smaller list.

I'll move this out to a README